requirements.txt

```
@@ -1,22 +1,18 @@
-Django==2.1.5
-argparse>=1.2.1
-ipaddress>=1.0.22
+# Django 4.2.19 is part of the Django 4.2 LTS and will reach end of life April 2026
+Django==4.2.19
```

Before the upgrade, `requirements.txt` was letting point releases of Django in - to allow any newer 1.11.x, it had: `Django>=1.11,<2.0`.
How about we accept newer 4.2.x with
```suggestion
Django>=4.2.19,<5.0
```

---

> Before the upgrade, `requirements.txt` was letting point releases of Django in - to allow any newer 1.11.x, it had: `Django>=1.11,<2.0`. How about we accept newer 4.2.x with

4.2.x is the LTS releases, so I don't think that adding `,<5.0` would be quite right. I think what we want is `Django>=4.2.19,<4.3`

---

Thanks for the feedback.

I went by what the previous rule was (`<2.0` for 1.11.x), but I'm not sure what the exact motivation behind that was.  From the announced Django versions, there won't be any 4.x past 4.2.x, so `<4.3` will work the same as `<5.0`.

I'm happy with either - just wanted to make it accept newer  4.2.x (as it is still a supported release and newer versions may come out).

`Django~=4.2.19` should work too - leaving specific syntax up to you.

---

Done